.github/renovate.json

```diff
@@ -1,7 +1,13 @@
 {
-	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
-	"extends": [
-		"github>jellyfin/.github//renovate-presets/nodejs",
-		":semanticCommitsDisabled"
-	]
-}
\ No newline at end of file
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>jellyfin/.github//renovate-presets/nodejs", ":semanticCommitsDisabled"],
+  "packageRules": [
+    {
+      "description": "Disable major updates for React",
+      "matchPackagePrefixes": ["react"],
+      "matchPackageNames": ["@types/react", "@types/react-helmet", "@types/react-router-dom"],
+      "matchUpdateTypes": "major",
+      "enabled": false
+    }
+  ]
```

Comment:
Do we **never** want to get React updates from Renovate or just skip React 18? Closing a Renovate update PR just skips a single specific version.

Reply:
I believe any major updates for React will require support in docusaurus first.